.github/workflows/pr.yml

```diff
@@ -20,18 +20,3 @@ jobs:
       - name: Build Package
         run: pnpm build
 
-      - name: Check if version has been updated
```

Comment:
This ignored the prerelease segment of the SemVer, and wasn't useful on this repo as it will always fail unless we follow a different versioning strategy. I tried a few alternatives but nothing worked well or was not maintained